changelog.d/13584.misc

```diff
@@ -0,0 +1 @@
+Add metrics to time how long it takes us to do backfill processing (`synapse_federation_backfill_processing_before_time_seconds`, `synapse_federation_backfill_processing_after_time_seconds`).
```

Comment:
Since https://github.com/matrix-org/synapse/pull/13535 hasn't shipped in a release, using the same changelog so it merges.